simulators/lumi/qiskit/README.md

```diff
@@ -0,0 +1,51 @@
+# Running Quantum programs with Qiskit Simulator on LUMI
+
+## Loading the qiskit module
+The qiskit module on LUMI can be loaded using the following command:
+
+```
+module use /appl/local/quantum/modulefiles
+module load qiskit
+```
+
+This will load the default version of qiskit along with other major Qiskit packages (Terra, Nature, Aer, etc.)
+
+## Run the quantum programs with sbatch
+You can edit a test.sh file to run quantum programs. (recommended)
```

Comment:
Can we use named code blocks as described [here](https://sphinx-book-theme.readthedocs.io/en/stable/content/pydata-content-blocks.html#code-blocks) e.g 

```
.. code-block:: sh
    :caption: test.sh

    ...
```